### ISSUE TITLE
Can you provide a simple demo script? 

### ISSUE THREAD

**Issue by Coronal-Halo** [zyh-uaiaaaa/Open-Set-FER#1]
Hi, could you please provide a simple demo script where a custom image is taken by the model to output an emotion label? Your Python notebook does not work due to multiple errors and missing pieces...

**Comment by zyh-uaiaaaa** [comment]
Sure, I’ll update in two days.

--------------原始邮件--------------
发件人："Coronal-Halo ***@***.***&gt;;
发送时间：2024年7月15日(星期一) 晚上6:45
收件人："zyh-uaiaaaa/Open-Set-FER" ***@***.***&gt;;
抄送："Subscribed ***@***.***&gt;;
主题：[zyh-uaiaaaa/Open-Set-FER] Can you provide a simple demo script?  (Issue #1)
-----------------------------------

 

 
Hi, could you please provide a simple demo script where a custom image is taken by the model to output an emotion label? Your Python notebook does not work.

**Comment by zyh-uaiaaaa** [comment]
In the open_set_detection_test.ipynb, a model is trained using the pseudo labels. The trained model is evaluated on the test set which contains both closed-set and open-set samples. We record the index of the closed-set and open-set samples in order to view the separation of their predictions using the trained model. A better separation means that the model is better for open-set recognition. You only need to download the pretrained ResNet-50 to start training. Could you provide the error information as I do not see the missing pieces.

**Comment by ChenJian7578** [comment]
你好，我想在开源的数据上进行测试，请问当我使用人脸检测模型得到人脸区域后，需要进行人脸矫正等预处理操作吗？或者可以提供一个完整的示例脚本，包括人脸检测的钱箱步骤。
